content/docs/start/data-and-model-versioning.md

```diff
@@ -62,8 +62,8 @@ $ git commit -m "Add raw data"
 
 ### 💡 Expand to see what happens under the hood.
 
-`dvc add` moved the data to the project's <abbr>cache</abbr>, and linked\* it
-back to the <abbr>workspace</abbr>.
+`dvc add` moved the data to the project's <abbr>cache</abbr>, and
+<abbr>linked</abbr> it back to the <abbr>workspace</abbr>.
```

Comment:
I'd leave this for the basic concepts PR. I'm not sure file linking is a basic concepts. Maybe

Reply:
I guess it can be! And I like this change. But it's not a copy edit so it prob belongs to a separate PR.